spec/recurly/plan_spec.rb

```diff
@@ -138,12 +138,117 @@
+            <USD>6000</USD>\
+          </setup_fee_in_cents>\
+          <setup_fee_performance_obligation_id>1</setup_fee_performance_obligation_id>\
+          <unit_amount_in_cents>\
+            <USD>7900</USD>\
+          </unit_amount_in_cents>\
+        </plan>
+      XML
+    end
+
+    it 'must serialize' do
+      plan.to_xml.must_equal expected_xml
+    end
+
+    it 'must have the RevRec attributes' do
+      plan.setup_fee_performance_obligation_id.must_equal('1')
+      plan.performance_obligation_id.must_equal('7uf')
+      plan.revenue_gl_account_id.must_equal('6t5')
+      plan.liability_gl_account_id.must_equal('8y7')
+    end
+  end
+
+  # describe 'base fee RevRec attributes' do
+  #   let(:base_fee_plan) {
+  #     Plan.new(
+  #       :plan_code                              => 'gold',
+  #       :name                                   => 'The Gold Plan',
+  #       :unit_amount_in_cents                   => 79_00,
+  #       :description                            => 'The Gold Plan is for folks who love gold.',
+  #       :setup_fee_in_cents                     => 60_00,
+  #       :plan_interval_length                   => 1,
+  #       :plan_interval_unit                     => 'months',
+  #       :pricing_model                          => 'fixed',
+  #       :setup_fee_performance_obligation_id    => '1',
+  #       :performance_obligation_id              => '7uf',
+  #       :revenue_gl_account_id                  => '6t5',
+  #       :liability_gl_account_id                => '8y7',
+  #     )
+  #   }
+
+  #   let(:expected_xml) do
+  #     <<~XML.gsub('  ', '').chomp
+  #       <plan>\
+  #         <description>The Gold Plan is for folks who love gold.</description>\
+  #         <liability_gl_account_id>8y7</liability_gl_account_id>\
+  #         <name>The Gold Plan</name>\
+  #         <performance_obligation_id>7uf</performance_obligation_id>\
+  #         <plan_code>gold</plan_code>\
+  #         <plan_interval_length>1</plan_interval_length>\
+  #         <plan_interval_unit>months</plan_interval_unit>\
+  #         <pricing_model>fixed</pricing_model>\
+  #         <revenue_gl_account_id>6t5</revenue_gl_account_id>\
+  #         <setup_fee_in_cents>\
+  #           <USD>6000</USD>\
+  #         </setup_fee_in_cents>\
+  #         <setup_fee_performance_obligation_id>1</setup_fee_performance_obligation_id>\
+  #         <unit_amount_in_cents>\
+  #           <USD>7900</USD>\
+  #         </unit_amount_in_cents>\
+  #       </plan>
+  #     XML
+  #   end
+
+  #   it 'must serialize' do
+  #     base_fee_plan.to_xml.must_equal expected_xml
+  #   end
+
+  #   it 'must have the RevRec attributes' do
+  #     base_fee_plan.setup_fee_performance_obligation_id.must_equal('1')
+  #     base_fee_plan.performance_obligation_id.must_equal('7uf')
+  #     base_fee_plan.revenue_gl_account_id.must_equal('6t5')
+  #     base_fee_plan.liability_gl_account_id.must_equal('8y7')
+  #   end
+  # end
+
```

Comment:
What's the plan for this commented lines?

Reply:
They were meant to be deleted, fixed now!